core/modules/qdisp/Executive.h

```diff
@@ -136,12 +149,20 @@ class Executive {
     std::shared_ptr<MessageStore> _messageStore; ///< MessageStore for logging
     XrdSsiService* _service; ///< RPC interface
     RequesterMap _requesters; ///< RequesterMap for results from submitted tasks
-    StatusMap _statuses; ///< Statuses of submitted tasks
+    JobStatusPtrMap _statuses; ///< Statuses of submitted tasks
+
+    /** Execution errors */
```

Comment:
O, but I try to follow LSST documentation standards: https://confluence.lsstcorp.org/display/LDMDG/Documentation+Standards#DocumentationStandards-RequiredDocumentationStyle
and they looks different from the previous standards used by Qserv. Shall we switch for new comment and create a low priority ticket to convert everything?